components/brave_wallet_ui/image/image.html

```diff
@@ -0,0 +1,105 @@
+<!-- Copyright (c) 2021 The Brave Authors. All rights reserved. -->
+<!-- This Source Code Form is subject to the terms of the Mozilla Public -->
+<!-- License, v. 2.0. If a copy of the MPL was not distributed with this file, -->
+<!-- You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Untrusted Image iframe</title>
+
+  <style>
```

Comment:
if you move this style to its own file, can you remove the `unsafe-inline` from https://github.com/brave/brave-core/pull/14535/files#diff-41dbb3aaf836430e94a9aba4d486bfa62c82a4d3058447d094ef20a443c3964eR49?

Reply:
Not sure how to load the css when it is placed in another file. What do I need to change in the build files?

Reply:
@josheleonard as mentioned above please un-inline at least the JS